utils/heightthresholding.py

```diff
@@ -62,7 +62,7 @@ def otsuthresholdgrainfinding(data, k):
     # # Invert mask to maks things below the membrane
     mask.grains_invert()
 
-    gwy.gwy_process_func_run("align_rows", data, gwy.RUN_IMMEDIATE)
```

Comment:
If this is a Gwyddion function (leftover code I know), have you added the "median_flatten" here as a reference to what it _should_ be doing when equated to our code?

Reply:
Shouldn't this be deleted if it's calling gwy, since AFAIK gwy is no longer able to be ran / loaded?

Reply:
This is an old file as its under `utils/` I think I made a comment on this too and we should definitely purge all these old things before releasing.

Reply:
Oh I see, I did a find and replace which is why this was edited, not sure if it breaks anything so I wasn't sure what to do with it.

Reply:
I think I'll leave this in with the updated function name just in case it helps with clarity when refactoring